test/SlowTests/Server/Documents/PeriodicBackup/PeriodicBackupSlowTests.cs

```diff
@@ -3923,6 +3930,13 @@ public async Task can_snapshot_and_restore_with_subscription(BackupType backupTy
                     }.Initialize();
 
                     subscriptionsConfig = await destination.Subscriptions.GetSubscriptionsAsync(0, 10);
+
+                    await WaitForValueAsync(async () =>
```

Comment:
This isn't needed.
We wait for the subscription state during the restore.

Reply:
Removed